pkgs/by-name/ze/zellij/package.nix

```diff
@@ -0,0 +1,99 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  stdenv,
+  installShellFiles,
+  pkg-config,
+  libiconv,
+  curl,
+  openssl,
+  DiskArbitration,
+  Foundation,
+  mandown,
+  zellij,
+  testers,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "zellij";
+  version = "0.41.1";
+
+  src = fetchFromGitHub {
+    owner = "zellij-org";
+    repo = "zellij";
+    rev = "v${version}";
+    hash = "sha256-EUoJHM0Jm0uFKFeHhtzon/ZRC615SHfYa1gr4RnCNBw=";
+  };
+
+  cargoHash = "sha256-rI3pa0dvC/OVJz8gzD1bM0Q+8OWwvGj+jGDEMSbSb2I=";
+
+  env.OPENSSL_NO_VENDOR = 1;
+
+  # Workaround for https://github.com/zellij-org/zellij/issues/3720
+  postPatch = ''
+    substituteInPlace zellij-utils/Cargo.toml \
+      --replace-fail 'isahc = "1.7.2"' 'isahc = { version = "1.7.2", default-features = false, features = ["http2", "text-decoding"] }'
+  '';
+
+  nativeBuildInputs = [
+    mandown
+    installShellFiles
+    pkg-config
+    (lib.getDev curl)
+  ];
+
+  buildInputs =
+    [
+      curl
+      openssl
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      libiconv
+      DiskArbitration
+      Foundation
+    ];
```

Comment:
I tested earlier that these can be entirely removed after https://github.com/NixOS/nixpkgs/pull/346043. I mention it now because since you are moving this package to by-name, removing these inputs also means you can remove the entry entirely from all-packages.nix.

Reply:
Ah right, ok, I'll do that!